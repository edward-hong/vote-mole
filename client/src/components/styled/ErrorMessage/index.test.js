import React from 'react'
import { render } from '@testing-library/react'

import ErrorMessage from '.'

it('renders correctly', () => {
	const {
		container: { firstChild: element },
	} = render(<ErrorMessage message="There's an error" />)

	expect(element).toMatchSnapshot()
})
