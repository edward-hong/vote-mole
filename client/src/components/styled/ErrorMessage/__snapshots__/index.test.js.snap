// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  color: #dc3545;
}

<div
  class="c0"
>
  There's an error
</div>
`;
